# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender_api/src/font.rs
# Commit: 793e9dfa4b9f
# Full Hash: 793e9dfa4b9fa40137cdf2a5d34c126d6cc9840d
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-02-29 04:04:56
# Regressor Bug: 1616995
# File Overlap Count: 1
# Description:
#   Bug 1616995 - patch 2 - Support vertical skew for upright-vertical fonts in webrender. r=lsalzman
#   
#   Depends on D63893
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D64746
# ==============================================================================

diff -r 88d3197f16c2 -r 793e9dfa4b9f gfx/wr/webrender_api/src/font.rs
--- a/gfx/wr/webrender_api/src/font.rs	Fri Feb 28 14:55:39 2020 +0000
+++ b/gfx/wr/webrender_api/src/font.rs	Fri Feb 28 15:36:19 2020 +0000
@@ -179,6 +179,7 @@
         const FLIP_X            = 1 << 5;
         const FLIP_Y            = 1 << 6;
         const SUBPIXEL_POSITION = 1 << 7;
+        const VERTICAL          = 1 << 8;
 
         // Windows flags
         const FORCE_GDI         = 1 << 16;
